firefox-ios/Client/Nimbus/NimbusSplashScreenFeatureLayer.swift

```diff
@@ -0,0 +1,21 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+protocol NimbusSplashScreenFeatureLayerProtocol {
```

Comment:
I followed what was done for the other features like `NimbusFirefoxSuggestFeatureLayerProtocol`. I also saw `NimbusFeatureFlagWithCustomOptionsID` having `searchBarPosition`, but felt that this way was cleaner than adding to that enum. 

Let me know if there's a better way in accessing this data, open to suggestions!

Reply:
No, this is the best way to do it. We should have layers for the different things. :)